Maths/AverageMean.js

```diff
@@ -13,11 +13,7 @@ const mean = (nums) => {
     throw new TypeError('Invalid Input')
   }
 
-  // This calculates the sum of all values in the 'nums' array using reduce method.
-  const sum = nums.reduce((sum, cur) => sum + cur, 0)
-
-  // Divide sum by the length of the 'nums' array.
-  return sum / nums.length
+  return nums.reduce((sum, cur) => sum + cur / nums.length, 0)
```

Comment:
```suggestion
  return nums.reduce((sum, cur) => sum + cur, 0) / nums.length
```

This is not quite the same as first summing, then dividing by the length; among other things, it is more costly, and a priori I don't see a reason why it would be more numerically stable either.

Reply:
agreed. first summing and then dividing the entire sum will definitely take less operations